src/components/observability/entities/alarm.ts

```diff
@@ -19,7 +20,7 @@ export default abstract class Alarm {
     extraConfigs: Required<
         Omit<AlarmExtraConfigs, 'parent' | 'snsTopicArns' | 'standardDeviation'>
     > &
-        Pick<AlarmExtraConfigs, 'snsTopicArns' | 'parent' | 'standardDeviation'>;
+        Pick<AlarmExtraConfigs, 'snsTopicArns' | 'parent'>;
```

Comment:
I think we have to remove the standardDeviation from the first part of the type definition as well.

```typescript
    extraConfigs: Required<
        Omit<AlarmExtraConfigs, 'parent' | 'snsTopicArns'>
    > &
        Pick<AlarmExtraConfigs, 'parent' | 'snsTopicArns'>;
 ```